src/components/common/organisms/ChangeLog.tsx

```diff
@@ -1,46 +1,70 @@
-import React from 'react';
-import { View, ScrollView, StyleSheet, useWindowDimensions } from 'react-native';
+import React, { useState } from 'react';
+import {
+  View,
+  ScrollView,
+  StyleSheet,
+} from 'react-native';
 import Button from '../atoms/Button';
 import Spinner from '../atoms/Spinner';
-import Colors from "../../../stylesheets/colors";
-import * as showdown from "showdown";
-import RenderHtml, { defaultSystemFonts } from "react-native-render-html"
-import moment from 'moment';
+import Colors from '../../../stylesheets/colors';
+import * as showdown from 'showdown';
+import moment from "moment";
+import RenderHtml, { defaultSystemFonts } from 'react-native-render-html';
 import { Typography } from 'mad-expo-core';
 
-
 const featureTitle = "What's new";
 const affirmText = 'OK';
 const converter = new showdown.Converter();
-const systemFonts = [...defaultSystemFonts, "Equinor-Regular", "Equinor-Medium"]
+const systemFonts = [
+  ...defaultSystemFonts,
+  'Equinor-Regular',
+  'Equinor-Medium',
+];
 
 const ChangeLog = (props: {
-    releaseNote: string;
-    fetching: boolean;
-    affirm: any;
-  }) => {
+  releaseNote: string;
+  fetching: boolean;
+  affirm: any;
+}) => {
+  const renderChangeLog = (release: any) => {
```

Comment:
Nitpicking, but should this item be typed?
This goes for most of the data we use in MEC i would assume.
It has the benefits of clearly defining a contract that our library needs our api to adhere to. Mapping from API responses to internal library data types also becomes more chill.

Reply:
added new type Release with named props